src/jquery.js

```diff
@@ -40,6 +40,89 @@ function init() {
   store.commit('app/setKeypressListener', () => keypressListener);
 }
 
+// returns true if bit `value` of `test` is true
+function hasBitsSet(test, value) {
+  return (test & (1 << value)) === Math.pow(2, value);
+}
+
+// check One-Shot Mod keycodes
```

Comment:
expand this comment please. basically explain you're normalizing out of order osm mods that were entered by hand potentially